crane/hooks/slack.py

```diff
@@ -63,14 +63,14 @@ def aslist(cls, obj):
 class Hook(Base):
     def __init__(self):
         self.token = settings["slack_token"]
-        self.slack_channel = settings["slack_channel"]
+        self.slack_channels = settings["slack_channels"]
         # The upcoming line is the most ridiculous, stupid, and effective hack I've ever written.
         # We create a link that has only a space as its link text, so it doesn't show up in Slack.
         # This allows us to store data in a fake URL, instead of needing a database or something.
         # Ridiculous.
         self.deployment_text = f"<{deployment.id}.com| >"
 
-        if self.token and self.slack_channel:
+        if self.token and len(self.slack_channels) > 0:
```

Comment:
Should be self.slack_channels only, will change that.
```suggestion
        if self.token and self.slack_channels:
```